.travis.yml

```diff
@@ -1,16 +1,48 @@
+dist: bionic
 language: python
 sudo: false
 cache:
+  pip: true
   directories:
   - eggs
-  - downloads
 python:
-  - 2.7
+  - "2.7"
 matrix:
+  include:
+    # - python: "2.7"
+    #   env: PLONE_VERSION=43
+    # - python: "2.7"
+    #   env: PLONE_VERSION=51
+    # - python: "2.7"
+      env: PLONE_VERSION=52
+    - python: "3.7"
+      env: PLONE_VERSION=52
+  sudo: true
   fast_finish: true
+before_install:
+  - sudo apt-get install -y firefox-geckodriver
```

Comment:
firefox is not needed, because we don't have robot tests.

Reply:
removed